docs/topics/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -0,0 +1,64 @@
+[#proc_deploying-an-application-to-openshift]
+= Deploying an Application to OpenShift
+
+.Prerequisites
+
+* The `oc` CLI client installed.
+* Maven installed.
+
+.Procedure
```

Comment:
It may be good to say this happens in the directory with the application, but I'll leave that to you.

Reply:
Thank you, that is important information. Let me add it there:)